harness/attributes/docker.yml

```diff
@@ -47,7 +47,7 @@ attributes:
 
     mysql:
       enabled: "= 'mysql' in @('app.services')"
-      image: mysql:5.7
+      image: mysql:8
```

Comment:
Maybe fine if not using php to talk to it, but this will need some extra parameters regarding authentication if you are.

Reply:
PHP wont be connecting to it (also appears PHP 7.4 resolved the issue)